docker/iml-cli-proxy.sh

```diff
@@ -2,5 +2,9 @@
 
 trailer=$(docker service ps -f 'name=iml_iml-manager-cli.1' iml_iml-manager-cli -q --no-trunc | head -n1)
 TTYOPT=$(tty -s && echo "-t")
+ENVOPT=""
+if [ -n "$RUST_LOG" ]; then
+	ENVOPT="-e RUST_LOG=$RUST_LOG"
+fi
 
-docker exec -i $TTYOPT iml_iml-manager-cli.1.$trailer iml "${@:1}"
+docker exec -i $TTYOPT $ENVOPT iml_iml-manager-cli.1.$trailer iml "${@:1}"
```

Comment:
Nit.
Since we use bash anyway, to avoid any chance for problem with spaces:

```bash

ENVOPTS=()
if [ -n "$RUST_LOG" ]; then
	ENVOPTS+=( -e "RUST_LOG=$RUST_LOG" )
fi
docker exec -i $TTYOPT "${ENVOPTS[@]}" iml_iml-manager-cli.1.$trailer iml "${@:1}"
```

Could also construct the entire docker command this way.